manager/burrow-mint/state/block_cache.go

```diff
@@ -224,8 +225,9 @@ func (cache *BlockCache) Sync() {
 		if removed {
 			removed := cache.backend.RemoveAccount([]byte(addrStr))
 			if !removed {
-				sanity.PanicCrisis(fmt.Sprintf("Could not remove account to be removed: %X", acc.Address))
+				sanity.PanicCrisis(fmt.Sprintf("Could not remove account to be removed: %X", addrStr))
```

Comment:
If we do get here better to not get a nil pointer error before the panic sanity if acc == nil